recipes/coinasl/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "1.4.3" %}
+
+package:
+  name: coinasl
+  version: {{ version }}
+
+source:
+  url: https://github.com/coin-or-tools/ThirdParty-ASL/archive/releases/{{ version }}.tar.gz
+  sha256: 0a4cfa7c6c5df72d8e1c4c3ce74633421f807115a679e85cb5569c41df9a9aaf
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage('coinasl', max_pin='x.x') }}
+
+requirements:
+  build:
+    - make  # [unix]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - make
+    - pkg-config
+    - wget
+  host:
+    - libblas
+    - libcblas
+    - liblapack
+
+test:
+  commands:
+    - test -f "${PREFIX}/lib/libcoinasl${SHLIB_EXT}"
+
+about:
+  home: http://www.coin-or.org/
+  license: EPL-1.0
+  # additional MIT license for ASL code
+  license_file: LICENSE.rst
```

Comment:
I think you can package more than one license file here: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#license-file

Reply:
indeed